### PR TITLE
Replace references to https://eosc-portal.eu/

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

The [EOSC Portal](https://eosc-portal.eu/about) redirects now to the [EOSC EU Node](https://open-science-cloud.ec.europa.eu/).

The [EOSC Marketplace](https://marketplace.eosc-portal.eu/) is no longer available.

This PR updates the references to both.

---

**Related issue :**

### CODE REVIEW

File: content/en/providers/joining/federated-resource-centre/_index.md [hunk 1]
```diff
@@ -181,8 +181,7 @@ International communities reach EGI through the following channels:
 - The EGI site where they can
   [request access to services](https://www.egi.eu/services/research/).
 - The [EGI-ACE Call for Use Cases](https://www.egi.eu/egi-ace-open-call/).
-- The [Marketplace](https://marketplace.eosc-portal.eu/) of the European Open
-  Science Cloud.
+- The [European Open Science Cloud](https://open-science-cloud.ec.europa.eu/).
```
Comment (enolfc) [inline content/en/providers/joining/federated-resource-centre/_index.md:184]: Is EGI easily reachable through this web? I would remove this one

Comment (sebastian-luna-valero) [reply content/en/providers/joining/federated-resource-centre/_index.md]: Agreed and done. Thanks!

**Review by gwarf — APPROVED** [review]
LGTM

**Review by enolfc — APPROVED** [review]
👍

**Pull request merged**